perl/lib/YAMLScript.pm

```diff
@@ -17,7 +17,7 @@ sub new {
 
 unless ( Alien::YAMLScript->exact_version($libyamlscript_version) ) {
     my $have = Alien::YAMLScript->version;
-    die "YAMLScript::FFI $VERSION requires Alien::YAMLScript libyamlscript version\n" .
+    die "YAMLScript $VERSION requires Alien::YAMLScript libyamlscript version\n" .
```

Comment:
I hadn't noticed it before, but this stutters a bit. Shouldn't it be enough to specify a required version of Alien::YAMLScript, without having to mention "libyamlscript" explicitly?

Reply:
I'm fine with just making sure `$YAMLScript::VERSION` matches `$Alien::YAMLScript::VERSION` and then having `Alien::YAMLScript` load an exact library version.